Components/Alerts/alerts.css

```diff
@@ -0,0 +1,52 @@
+:root{
+    --alert_error : rgb(235, 72, 72); 
+    --alert_success : rgb(50, 189, 50); 
+    --alert_warning : rgb(236, 178, 70); 
+    --alert_primary : skyblue;  
+}
+
+.alert_container{
+  display: flex;  
```

Comment:
A suggestion here:-
You can make a utility class of `display: flex` in just one CSS file( which you use almost everywhere ) and reuse it from there rather than repeating this property in every component

Reply:
sure kirti...will change accordingly